app/Decsys/Models/Wordlist/UserWordlist.cs

```diff
@@ -0,0 +1,12 @@
+using Decsys.Data.Entities;
+namespace Decsys.Models.Wordlist;
+
+public class UserWordlist
```

Comment:
This needs a base Entity with everything except WordlistRules, and then Mongo and LiteDb entities which inherit from BaseUserWordlist and have the correct Rules property.